src/test/java/janggi/service/JanggiServiceTest.java

```diff
@@ -0,0 +1,269 @@
+package janggi.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import janggi.dao.BoardDAO;
+import janggi.dao.FakeBoardDAO;
+import janggi.dao.FakeGameRoomDAO;
+import janggi.dao.GameRoomDAO;
+import janggi.domain.Board;
+import janggi.domain.GameRoom;
+import janggi.domain.Team;
+import janggi.domain.move.Position;
+import janggi.dto.TeamHorseElephantPositionDto;
+import janggi.util.BoardFixture;
+import janggi.view.HorseElephantPosition;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class JanggiServiceTest {
+
+    GameRoomDAO gameRoomDAO;
+    BoardDAO boardDAO;
+    JanggiService janggiService;
```

Comment:
테스트코드에서도 캡슐화를 지켜 주세요 😄 

```suggestion
    private GameRoomDAO gameRoomDAO;
    private BoardDAO boardDAO;
    private JanggiService janggiService;
```

Reply:
수정하였습니다~

[refactor: 테스트 접근 제한자 추가](https://github.com/woowacourse/java-janggi/pull/90/commits/1125d68fadc802a810fe94ab4ede52f806e2c488)